docs/clusters/gcp/install.md

```diff
@@ -74,8 +74,6 @@ The docker images used by the Cortex cluster can also be overridden, although th
 image_operator: quay.io/cortexlabs/operator:0.27.0
 image_manager: quay.io/cortexlabs/manager:0.27.0
 image_downloader: quay.io/cortexlabs/downloader:0.27.0
-image_statsd: quay.io/cortexlabs/statsd:0.27.0
 image_istio_proxy: quay.io/cortexlabs/istio-proxy:0.27.0
 image_istio_pilot: quay.io/cortexlabs/istio-pilot:0.27.0
-image_pause: quay.io/cortexlabs/pause:0.27.0
```

Comment:
this is indeed a mistake but it should instead be replaced by:

`image_google_pause: quay.io/cortexlabs/google-pause:0.27.0`